### PR TITLE
Refactor Store tests

### PR DESCRIPTION
**Pull request by takahirom**
## Overview (Required)
- The store's test is not in order, just checking if you can take the value and get it.
- This issue is needed for creating test issues

### CODE REVIEW

File: feature/announcement/src/main/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStore.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ class AnnouncementStore @Inject constructor(
     val loadingState: LiveData<LoadingState> = dispatcher
         .subscribe<Action.AnnouncementLoadingStateChanged>()
         .map { it.loadingState }
-        .toLiveData(LoadingState.LOADING)
```
Comment (jmatsu) [inline feature/announcement/src/main/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStore.kt]: 👍

File: feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt [hunk 2]
```diff
@@ -31,30 +31,26 @@ class AnnouncementStoreTest {
         val observer = mockk<(LoadingState?) -> Unit>(relaxed = true)
 
         announcementStore.loadingState.changedForever(observer)
+        coVerify { observer(LoadingState.INITIALIZED) }
```
Comment (jmatsu) [inline feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt]: I'm curious so let me ask one question, is this `coVerify` needed? Didn't `verify` work then?

Comment (takahirom) [reply feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt]: I will check that👍

Comment (takahirom) [reply feature/announcement/src/test/java/io/github/droidkaigi/confsched2019/announcement/ui/store/AnnouncementStoreTest.kt]: I confirm that. It is not needed. So I changed to verify! Thanks 👍

**Pull request merged**